app/mailers/helps_mailer.rb

```diff
@@ -0,0 +1,16 @@
+# typed: false
+# frozen_string_literal: true
+
+# The endpoint for the help modal
+class HelpsMailer < ApplicationMailer
+  def send_jira(email, subject, body)
+    @email = email
```

Comment:
do we need these as ivars?

Reply:
Only if we want them to automagically populate a mail layout, but we aren't using one at this point?

Reply:
If we're not using a layout, let's not have ivars

Reply:
Fixed